google-cloud-spanner/src/test/java/com/google/cloud/spanner/SessionPoolTest.java

```diff
@@ -51,7 +51,6 @@
 import com.google.cloud.spanner.MetricRegistryTestUtils.PointWithFunction;
```

Comment:
Verify other tests in this class. On whether they are working as intended or not.